CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 
 <!-- Your comment below this -->
 - Simplified Bitrise repo slug lookup, fixing SSH URL parsing in BitBucketServer - [@rogerluan]
+- Changed `CHANGELOG.md` merge strategy to union to avoid merge conflicts - [@rogerluan]
```

Comment:
I added a changelog entry for https://github.com/danger/danger-js/pull/1116#issuecomment-781369623 lemme know if this should be reverted either because it's a meta change or doesn't belong to this PR 🤗

Reply:
I usually give the changelog notes a clean up before a release, it can stay (because it's not worth extra work undoing) but I treat the changelog as user-facing. Conceptually a user of danger shouldn't care too much about our merging strat in PRs.

Reply:
Ah gotcha! Noted for future events ✅ thanks!! 💪